### PR TITLE
[TASK] Overhaul HMENU Language page

### PR DESCRIPTION
**Pull request by linawolf**
* Use confvals
* Correct indentation
* Overhaul example

releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ContentObjects/Hmenu/Language.rst [hunk 1]
```diff
@@ -1,90 +1,78 @@
...
+..  seealso::
    For a language menu styled by Fluid see
    :ref:`LanguageMenuProcessor <LanguageMenuProcessor>`.
 
 Note on item states
 ====================
 
-When "TSFE->sys\_language\_uid" matches the sys\_language uid for an
-element the state is set to "ACT", otherwise "NO". However, if a page
+When `TSFE->sys_language_uid` matches the UID of an
```
Comment (brotkrueml) [inline Documentation/ContentObjects/Hmenu/Language.rst:68]: TSFE->sys_language_uid was deprecated in 9.4 and removed with 10.0:
https://github.com/TYPO3/typo3/blob/9.5/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php#L603-L611

So, this sentence should be adjusted.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/ContentObjects/Hmenu/Language.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ContentObjects/Hmenu/Language.rst:68]: You have any idea, how it works now?

Comment (brotkrueml) [reply Documentation/ContentObjects/Hmenu/Language.rst:68]: Nope, maybe add a todo and an issue.

Comment (linawolf) [reply Documentation/ContentObjects/Hmenu/Language.rst]: maybe like this

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**